src/code/audio/code_8019AF00.c

```diff
@@ -67,8 +61,30 @@ typedef struct {
 } FreqLerp; // size = 0x10
 
 s32 AudioOcarina_MemoryGameNextNote(void);
+
+void AudioSfx_ProcessSfxSettings(void);
+void func_8019FEDC(void);
+void Audio_StepFreqLerp(FreqLerp* lerp);
+
 s32 Audio_SetGanonsTowerBgmVolume(u8 targetVolume);
```

Comment:
Oh, I thought this would be a RiverSound. Is it even used in MM?

Reply:
It's from OoT's RiverSound but was completely dropped in MM. The code is still here but goes completely unused (there's a lot of unused code remnant from OoT's RiverSound). It was dropped in favor of an overhaul that is `ObjSound` which hasn't been PR'd yet.

Reply:
I suppose it comes down to whether it has specific scene checks and audio output. If not, it should probably be named a less specifically?

Reply:
It only applies it through riversound params in OoT, but it doesn't really work for anything other than Ganon's Tower. It's designed specifically around progression of 8 stages which match to Ganon's Tower Climb:
(`u8 sGanonsTowerLevelsVol[8] = { 127, 80, 75, 73, 70, 68, 65, 60 };`)
and has a few ioPorts that are paired specifically to Ganon's Organ theme:
` Audio_SeqCmd8(SEQ_PLAYER_BGM_MAIN, 4, 15, lowPassFilterCutoff);`
`gAudioContext.seqPlayers[SEQ_PLAYER_BGM_MAIN].channels[channelIdx]->soundScriptIO[5]`

i.e. this code is designed very specifically around Ganon's Tower, and doesn't really work for anything else